### ISSUE TITLE
Show 3 images of a URL (expected, actual, diff) one at a time

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/VisualTestingByImagemagickInKatalonStudio#28]
See the following screenshot. This shows how 2 screenshot images of a URL is shown:

Characters are displayed too small.
I can not read anything!

![toosmallcharacterscannotreadanything](https://user-images.githubusercontent.com/4467417/87745594-509e7e00-c829-11ea-9721-817c8f2d2b0f.png)

**Comment by kazurayam** [comment]
How about using Bootstrap 4 Nav component.
https://getbootstrap.com/docs/4.0/components/navbar/

Lets have 3 tabs:

| Expected | Diff | Actual |

In each Tabs, each images are displaed with `class="img-fluid d-block mx-auto"`; possibly images are rendered at its original width.

**Comment by kazurayam** [comment]
Using Nav component means that we no longer use Carousel.

**Comment by kazurayam** [comment]
There is "Bootstrap Tabs Crousel"

https://www.jqueryscript.net/demo/bootstrap-tabs-carousel/

**Comment by kazurayam** [comment]
I tried Bootstrap 4 Navs with TAB format. Its works fine. But I realised that my approach is not straight to the point.
 
The real problem is that VisualTestingInKatalonStudio ver 1.21.1 displays the Expected image and the Actual image at a time allocating 50% width for each, consequentlyAnd I can display 3 images separately at a time. Consequently 2 images are scaled down and get hard to read. So, why not display the Expected image on a view and then the Acutal image on another view; separately allocating 100% width for each.

I can stick to Carousel format.
